src/components/helpers/hashtag_buildthelist.ts

```diff
@@ -1,4 +1,4 @@
-export function hashtagBuildTheList(item: number) {
+export function hashtagBuildTheList(item: number = 999) {
```

Comment:
I'm going to guess this is solving an "undefined" type error by giving it a pretty high (but not TOO unreasonably so default value?

Reply:
its my way of having it be backwards compatible for how it works in choc sauce, but allows me to pass a param that outputs _all_ the list for waffle cone purposes. So now the default passes the whole list instead of one item, and then if you ask it to pass one item (like used to be the only option) it will still do that. Down the line when we launch for real and have real users we may find out users actually have more than 999 rss feeds, in which case we'll need to update this